backend/census_historical_migration/test_secondary_auditors.py

```diff
@@ -0,0 +1,32 @@
+from django.conf import settings

```

Comment:
A different file name of 'test_secondary_auditor_xforms.py' might match the rest of the test files.

Reply:
Done